Content.Server/Radio/EntitySystems/RadioDeviceSystem.cs

```diff
@@ -218,7 +218,7 @@ private void OnReceiveRadio(EntityUid uid, RadioSpeakerComponent component, ref
             ("originalName", nameEv.VoiceName));
 
         // log to chat so people can identity the speaker/source, but avoid clogging ghost chat if there are many radios
-        _chat.TrySendInGameICMessage(uid, args.Message, InGameICChatType.Whisper, ChatTransmitRange.GhostRangeLimit, nameOverride: name, checkRadioPrefix: false);
+        _chat.TrySendInGameICMessage(uid, args.Message, (InGameICChatType) component.ChatType, ChatTransmitRange.GhostRangeLimit, nameOverride: name, checkRadioPrefix: false);
```

Comment:
```suggestion
        _chat.TrySendInGameICMessage(uid, args.Message, component.ChatType, ChatTransmitRange.GhostRangeLimit, nameOverride: name, checkRadioPrefix: false);
```
No cast needed if we're using the actual type.